src/components/blocks/InformationBlock/InformationBlock.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import InformationBlock from './InformationBlock';
+import type { InformationBlockProps } from './InformationBlock';
+
+describe('InformationBlock', () => {
+  describe("with a 'main' variant", () => {
+    it('renders with given text', () => {
+      const args: InformationBlockProps = {
+        variant: 'main',
+        data: [
+          {
+            id: '134',
+            value: 'foo',
+            label: "foo's label",
+          },
+        ],
+        header: 'this is the header',
+        icon: 'Icon',
+      };
+
+      render(<InformationBlock {...args} />);
+
+      const content = screen.getByText("foo's label");
+
+      expect(content).toBeInTheDocument();
+    });
+  });
+
+  describe("with a 'regular' variant", () => {
+    it('renders with given text', () => {
+      const args: InformationBlockProps = {
+        variant: 'regular',
+        data: [
+          {
+            id: '134',
+            value: 'foo',
+            label: "foo's label",
+          },
+        ],
+        header: 'this is the header',
+        icon: 'Icon',
+      };
+
+      render(<InformationBlock {...args} />);
+
+      const content = screen.getByText("foo's label");
```

Comment:
Same here, let's test `regular` class

For testing text you can add additional test case which will test if text is there